CollapseLauncher/Classes/CachesManagement/Honkai/Check.cs

```diff
@@ -71,52 +72,54 @@ private void CheckUnusedAssets(List<CacheAsset> assetIndex, List<CacheAsset> ret
             {
                 string filePath = fileInfo.FullName;
 
-                if (!filePath.Contains("output_log", StringComparison.OrdinalIgnoreCase)
-                 && !filePath.Contains("Crashes", StringComparison.OrdinalIgnoreCase)
-                 && !filePath.Contains("Verify.txt", StringComparison.OrdinalIgnoreCase)
-                 && !filePath.Contains("APM", StringComparison.OrdinalIgnoreCase)
-                 && !filePath.Contains("FBData", StringComparison.OrdinalIgnoreCase)
-                 && !filePath.Contains("asb.dat", StringComparison.OrdinalIgnoreCase)
-                 && !assetIndex.Exists(x => x.ConcatPath == fileInfo.FullName))
+                if (filePath.Contains("output_log",    StringComparison.OrdinalIgnoreCase)
```

Comment:
Can this string comparator uses HashSet instead?

Reply:
Even better, we can use ``SearchValues<string>`` instead for more efficient and performance result